packages/keystone/errorsUtils/safeFormatError.js

```
@@ -0,0 +1,305 @@
+        _updateExtensionsForKnownErrorCases(result, extensions, originalError)
+
+        // NOTE(pahaz): KeystoneJS v5 compatibility! We already have lots of test with that names.
+        // We need to use more specific name instead of the 'GraphQLError'.
+        // If you want to change it you probably want to change API compatibility.
+        if (originalError && originalError?.name?.toLowerCase() !== 'error') {
+            result.name = originalError.name
+        }
+
+        // NOTE(pahaz): KeystoneJS v5 data field from errors
+        // TODO(pahaz): drop this undocumented backward compatibility ...
+        if (originalError && originalError?.data && !result.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)